.editorconfig

```diff
@@ -0,0 +1,52 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# TS/JS-Files
+[*.{ts,js}]
+indent_size = 2
```

Comment:
not sure we have ts/js files at any point in time, but still prefer indent size 4 for all files

Reply:
just used the file from core, will make the adjustments as you noted.